framework/wazuh/core/cluster/utils.py

```diff
@@ -557,7 +557,12 @@ def raise_if_exc(result: object) -> None:
         raise result
 
 
-def init_auth_worker():
-    """Set authentication pool worker to ignore SIGINT signals to avoid 
-    throwing exceptions when shutting down the API in foreground mode."""
+def spawn_authentication_worker():
```

Comment:
Given that the three processes pool workers are initialized the same way, we could have a single function here or in `api/api/utils.py` that takes the process name as a parameter and reuse it in `api/scripts/wazuh_apid.py` and `framework/wazuh/core/cluster/dapi/dapi.py` instead of having 3 different ones.

```suggestion
def init_pool_worker(process_name: str) -> None:
```

It can be used like this

```
initializer=partial(api.utils.init_pool_worker, process_name)
```

Reply:
I agree with that, but we need to keep the single function on the framework side, maybe in `pyDaemonModule`, and import it in `api/scripts/wazuh_apid.py` and `framework/wazuh/core/cluster/dapi/dapi.py`

Reply:
Fixed at c49f5acd6de33ff8ff5502009ef8b4ea7b7d804a